Networking/Sources/Networking/Requests/FirebaseLoginRequest.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+
+// MARK: - FirebaseLoginResponse
+public struct FirebaseLoginResponse: Codable {
+    let kind, localID, email, displayName: String
+    let idToken: String
+    let registered: Bool
+    let refreshToken, expiresIn: String
+}
+
+// MARK: - FirebaseLoginRequest
+public struct FirebaseLoginRequest: RequestType {
+    public typealias ResponseType = FirebaseLoginResponse
+    
+    private let email: String
+    private let password: String
+    
+    public init(email: String, password: String) {
+        self.email = email
+        self.password = password
+    }
+    
+    public var baseUrl: URL { Constants.firebaseAuth }
+    public var path: String { "AIzaSyB0UczrurqM1STyI8tvx4QZVTyQVw4UJ7Q" }
+    public var method: String = "POST"
```

Comment:
is it right to pass the apiKey for the `path`
@ahmdmhasn

Reply:
And I have actually written the Unit tests last week. I've been waiting for your approval 😅

Reply:
Please unit test this clss to make sure the parameters and other values are sent as expected